qt-models/diveplannermodel.cpp

```diff
@@ -108,6 +108,7 @@ void DivePlannerPointsModel::loadFromDive(dive *d)
 	int cylinderid = 0;
 	last_sp.mbar = 0;
 	for (int i = 0; i < plansamples - 1; i++) {
+		if ((dc->last_manual_time.seconds) && (lasttime.seconds >= dc->last_manual_time.seconds)) break;
```

Comment:
no need for the extra brackets around the conditions. use `if (A && B)` instead of `if ((A) && (B))`.